### PR TITLE
style: change font to ubuntu

### PR DESCRIPTION
**Pull request by jcstein**
## Previews
Screenshot 2024-05-16 at 3 05 00 PM
Screenshot 2024-05-16 at 3 05 05 PM

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

## Summary by CodeRabbit

- **New Features**
  - Integrated Google Fonts for improved typography.
  - Updated the app's fonts to use Ubuntu and Ubuntu Mono.

- **Improvements**
  - Enhanced font styling for a more consistent and appealing look.

- **Removals**
  - Removed an outdated sidebar link related to integrating Range with rollup.

### CODE REVIEW

**Comment by yarikbratashchuk** [comment]
@jcstein, shall we make it a pr, update branch and merge it?

**Comment by jcstein** [comment]
I'll update branch. I think this requires a little more weigh in, but we should check with @Manav-Aggarwal @gupadhyaya @tzdybal @tuxcanfly @MSevey to see what they think

File: .vitepress/config.ts [hunk 1]
```diff
@@ -106,9 +106,25 @@ export default withMermaid({
...
+      'link',
+      { rel: 'preconnect', href: 'https://fonts.googleapis.com' }
+    ],
+    [
+      'link',
+      { rel: 'preconnect', href: 'https://fonts.gstatic.com', crossorigin: '' }
+    ],
+    [
+      'link',
+      { href: 'https://fonts.googleapis.com/css2?family=Ubuntu+Mono:ital,wght@0,400;0,700;1,400;1,700&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap', rel: 'stylesheet' }
```
Comment (jcstein) [inline .vitepress/config.ts:122]:
```suggestion
      { href: 'https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,400;0,700;1,400;1,700&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap', rel: 'stylesheet' }
```

**Comment by yarikbratashchuk** [comment]
i'm good with that

**Review by yarikbratashchuk — APPROVED** [review]

**Comment by jcstein** [comment]
closing after feedback at onsite

**Pull request closed without merging**